docs/rules/computed-property-return.md

```diff
@@ -0,0 +1,44 @@
+# Enforces that a return statement is present in computed property (computed-property-return)
+
+## :book: Rule Details
+
+This rule enforces that a `return` statement is present in `computed` properties.
+
+:-1: Examples of **incorrect** code for this rule:
+
+```js
+export default {
+  computed: {
+    foo () {
+    },
+    bar: function () {
+    }
+  }
+}
+```
+
+:+1: Examples of **correct** code for this rule:
+
+```js
+export default {
+  computed: {
+    foo () {
+      return true
+    },
+    bar: function () {
+      return false
+    }
+  }
+}
+```
+
+## :wrench: Options
+
+This rule has an object option:
+- `"treatUndefinedAsUnspecified"`: `true` (default) allows implicitly returning undefined with a `return;` statement.
```

Comment:
I think we should force to return something explicitly, thus in my opinion `return;` should also trigger an error and this option should not be necessary.
We could consider two types of messages:
- There is no `return` statement inside `foo` computed property
- There is an incomplete `return` statement inside `foo` computed property
 What do you think @armano2 @chrisvfritz ?